<?xml version="1.0"?>
<launch>
    <arg name="demo" default="false" />
    <group ns="handover" >
    <include file="$(find bimanual_handover)/launch/pc_filter.launch" >
        <arg name="demo_pc" value="$(arg demo)" />
    </include>
    <node name="init_gripper_stub" pkg="bimanual_handover" type="init_gripper_stub.py" output="screen" />
    <node name="process_pc_stub" pkg="bimanual_handover" type="process_pc_stub.py" output="screen" />
    <node name="grasp_stub" pkg="bimanual_handover" type="grasp_stub.py" output="screen" />
    <node name="capture_pc" pkg="bimanual_handover" type="capture_pc.py" output="screen" args="$(arg demo)"/>
    <node name="close_contact_mover" pkg="bimanual_handover" type="close_contact_mover.py" output="screen" />
    <node name="robot_setup_mover" pkg="bimanual_handover" type="robot_setup_mover.py" output="screen" >
    </node>
    <node name="finish_handover" pkg="bimanual_handover" type="finish_handover.py" output="screen" >
        <remap from="move_group" to="/move_group" />
    </node>
    <node name="handover_controller" pkg="bimanual_handover" type="handover_controller.py" args="full" output="screen" />
    </group>
</launch>
